requirements/edx/base.txt

```diff
@@ -35,6 +35,7 @@ django-threaded-multihost==1.4-1
 django-method-override==0.1.0
 djangorestframework==2.3.14
 django==1.4.18
+elasticsearch<1.0.0
```

Comment:
Will do - however, this is what is recommended by elasticsearch - let me know if you want me to change back (http://elasticsearch-py.readthedocs.org/en/master/)